contracts/SecurityTokenRegistry.sol

```diff
@@ -203,6 +207,19 @@ contract SecurityTokenRegistry is ISecurityTokenRegistry, EternalStorage {
         set(POLYTOKEN, IPolymathRegistry(polymathRegistry).getAddress("PolyToken"));
     }
 
+    /**
+     * @notice Set the getter contract address
+     * @param _getterContract Address of the contract  
+     */
+    function setGetterRegistry(address _getterContract) public onlyOwner {
+        require(_getterContract != address(0));
+        set(Encoder.getKey("STRGetter"), _getterContract);
```

Comment:
We can hardcode the STRGetter key.

Reply:
Why? I am asking to hardcode just the key `Encoder.getKey("STRGetter")`
Like we have hardcoded other keys.
Just to save the computation required to calculate the key.

Reply:
Ohh My bad, I was thinking you are talking about the address.